identity-service/sequelize/migrations/20220809211742-add_supporter_dethroned.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface) => {
+    return queryInterface.sequelize.query(
+      `ALTER TYPE "enum_SolanaNotifications_type" ADD VALUE 'SupporterDethroned'`
+    )
+  },
+
+  down: (queryInterface, Sequelize) => {
+    return queryInterface.sequelize.transaction(async (transaction) => {
+
+      // Recreate the old enum:
+
+      // Create a new enum
+      await queryInterface.sequelize.query(`
+        CREATE TYPE "enum_SolanaNotifications_type_new"
+        AS ENUM ('ChallengeReward', 'MilestoneListen', 'TipSend', 'TipReceive', 'Reaction', 'SupporterRankUp', 'SupportingRankUp');
+      `, { transaction })
+
+      // Set the column to the new enum
+      await queryInterface.sequelize.query(`
```

Comment:
For my own understanding, what happens to rows with `SupporterDethroned` when this swap happens?

Reply:
Ooh good question, this probably wouldn't work until those rows are dropped. Think this is probably a bug in many down migrations that add enum cases